src/main/java/org/reactome/addlinks/fileprocessors/HmdbMetabolitesFileProcessor.java

```diff
@@ -132,7 +130,7 @@ public HmdbMetabolitesFileProcessor()
 	
 	private void transformXmlToTsv(String inputFilename, String outfileName)
 	{
-		
+		System.setProperty("javax.xml.transform.TransformerFactory", "com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl");
```

Comment:
Set the two strings here as final variables?

Reply:
Unlikely they will ever be used anywhere else.

Reply:
Fair point